packages/fxa-settings/package.json

```diff
@@ -149,6 +149,6 @@
     "storybook-addon-rtl": "^0.5.0",
     "style-loader": "^1.3.0",
     "tailwindcss": "^3.3.1",
-    "webpack": "^4.43.0"
+    "webpack": "^5.84.1"
```

Comment:
Interesting that `fxa-settings` already had a webpack dep when `fxa-payments-server` does not. 🤔

Reply:
That was my fault with accidentally editing the wrong set of deps (dependencies vs devDependencies).  It's been fixed in a newer commit.

Reply:
eh not a silly question at all.  I don't quite understand why it was originally added either.

I think one effect this particular change has is that this newer version will be used by react-scripts since react-scripts 5's package.json specifies its webpack dep with a `^`.  iirc at least one of the deps requires a very new version of webpack 5.  Of course we could use `resolutions` in the root package.json to achieve the same thing if that's the _only_ reason for this being here.

In other words, I don't know for certain until I try deleting that line which I haven't done so at the time of writing.  😀

Reply:
Yeah I don't think this explicit dep is necessary.  I'll file a follow-up to see which of the Webpack deps we can remove.  Thanks for bring it up.